<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="ListView" name="listView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QListView" name="listView">
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Open">
    <property name="title">
     <string>&amp;Open</string>
    </property>
    <addaction name="actionPointCloud_Directory"/>
    <addaction name="actionAnnotation_File"/>
   </widget>
   <addaction name="menu_Open"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPointCloud_Directory">
   <property name="text">
    <string>Open &amp;PointCloud Directory</string>
   </property>
  </action>
  <action name="actionAnnotation_File">
   <property name="text">
    <string>&amp;Load Annotation File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListView</class>
   <extends>QListView</extends>
   <header>listview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
